cmd/go-filecoin/chain_daemon_test.go

```diff
@@ -2,34 +2,39 @@ package commands_test
 
```

Comment:
Rename the files to x_integration_test when they no longer use NewDaemon, or maybe x_command_test when explicitly testing the CLI. (though only in this PR if you can do so without clobbering diffs)

Reply:
Good idea.  I'll follow on with one file mutation PR at the end of all of the outstanding test PRs to avoid rebase difficulties and bad diffs.